plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/3dtiles/BuildingColor/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Radio } from "@web/sharedComponents";
+import { styled } from "@web/theme";
+
+import { BaseFieldProps } from "../../types";
+
+import useHooks from "./hooks";
+
+const BuildingColor: React.FC<BaseFieldProps<"buildingColor">> = ({
+  value,
+  dataID,
+  editMode,
+  onUpdate,
+}) => {
+  const { options, floods, handleUpdateColorType } = useHooks({
+    value,
+    dataID,
+    onUpdate,
+  });
+
+  return editMode ? null : (
+    <Radio.Group onChange={handleUpdateColorType} value={options.colorType} defaultValue="none">
+      <StyledRadio value="none">
+        <Label>色分けなし</Label>
+      </StyledRadio>
+      <StyledRadio value="height">
+        <Label>高さによる塗分け</Label>
+      </StyledRadio>
+      <StyledRadio value="purpose">
+        <Label>用途による塗分け</Label>
+      </StyledRadio>
+      <StyledRadio value="structure">
+        <Label>建物構造による塗分け</Label>
+      </StyledRadio>
+      <StyledRadio value="fireproof">
+        <Label>耐火構造種別による塗分け</Label>
+      </StyledRadio>
```

Comment:
I don't think all 3d tiles that PLATEAU has will have every single type. I could be wrong though. Is this just temporarily hardcoded?

Reply:
Oops... I didn't notice it. I will fix it.